spec/database_cleaner/base_spec.rb

```diff
@@ -231,23 +95,29 @@ module DatabaseCleaner
     end
 
     describe "clean_with" do
-      # FIXME hacky null strategy
-      # because you can't pass a NullStrategy to #clean_with
+      subject { described_class.new(:active_record) }
```

Comment:
I know this to make the existing specs work, but if you're modernising here, we, the rspec team, recommend you at least use named `subject`, and we don't really recommend using `described_class` (as it requires constants to be loaded before use) but that might be something for a later PR.

Reply:
TIL about named subjects! :smiley: I'll give that a go.

Also, if you ever want to make a pass through the test suites, they all need a lot of work! A decade of being tied to the 2010-era gem ecosystem, while the world moves on, has necessarily yielded a paradigm of testing implementation, not behavior. :/ I've already done some work on the DC::AR adapter gem's suite, but once this PR lands, we're finally free to modernize things!

Reply:
Okay, I've pushed a new commit that names all custom subjects, and removes usage of `described_class`, as well as a number of other cleanups. WDYT?